packages/create-sitecore-jss/src/templates/nextjs/scripts/config/plugins/fallback.ts

```diff
@@ -0,0 +1,19 @@
+import { ConfigPlugin, JssConfig } from '..';
+
+/**
+ * This config will set fallback values for properties that were left empty
+ * If neither env, nor other places had a proper value, this will ensure a fallback is set
+ */
+class ComputedPlugin implements ConfigPlugin {
+  // should always comes last
+  order = 99;
+
+  async exec(config: JssConfig) {
+    return Object.assign({}, config, {
+      defaultLanguage: config.defaultLanguage || 'en',
+      sitecoreApiKey: config.sitecoreApiKey || 'no-api-key-set'
```

Comment:
I know we had 'no-api-key-set' before, but now questioning if there's really any reason to have this? i.e. wouldn't an empty string result in the same "invalid API key" error when making requests in the app? Fine if we want to leave it too.

Reply:
That's a good point. Technically, we'd have the same result with empty API key before all these changes too. I think we were using 'no-api-key-set' value for better visibility and clarity when error occurs?

Reply:
Yeah, let's just leave it for now.